config/fedora.yml

```diff
@@ -0,0 +1,20 @@
+development:
+  user: fedoraAdmin
+  password: fedoraAdmin
+  url: http://localhost:8984/rest
+  base_path: /dev
+test:
+  user: fedoraAdmin
+  password: fedoraAdmin
+  url: http://localhost:8986/rest
+  base_path: /test
+production:
+  user: <%= ENV["PLUM_FEDORA_USER"] || "fedoraAdmin" %>
```

Comment:
Maybe we should file a Valkyrie ticket for that?  Seems strange to require a Fedora configuration file when no Fedora adapter is configured...